src/locales/en.json

```diff
@@ -9,6 +9,7 @@
     "identity": {
       "title": "Verify your identity",
       "hint": "Select the type of document you would like to upload",
+      "list_label": "Document types",
```

Comment:
This translation is missing from `/es.json`

Reply:
Will remove this definition here as it is unused now (and a duplicated label) since I have moved the ARIA list labels under the accessibility map.